KomaMRICore/src/datatypes/sequence/Delay.jl

```diff
@@ -1,10 +1,7 @@
 """
     delay = Delay(T)
 
-The Delay struct. The input delay time `T` must be non-negative.
-
-!!! note
-    This struct is meant to add delays to a sequence struct.
+The Delay struct.
```

Comment:
The wording "**The** Delay struct" is weird. I would explain better that is a special type used to add a delay to a sequence (mostly by doing seq+Delay(T)).

Reply:
Edited!